nobrainer/bayesian_utils.py

```diff
@@ -11,25 +14,89 @@
 tfd = tfp.distributions
 
 
-def default_mean_field_normal_fn(
-    is_singular=False,
-    loc_initializer=tf.keras.initializers.he_normal(),
+def default_loc_scale_fn(
+    is_singular=True,
+    loc_initializer=tf1.initializers.random_normal(stddev=0.1),
     untransformed_scale_initializer=tf1.initializers.random_normal(
         mean=-3.0, stddev=0.1
     ),
```

Comment:
is tf1 being used for compatibility. in that case perhaps this could be set in a script initialization, and this can default to the current package.

Reply:
right, corrected that part